types/jquery/index.d.ts

```diff
@@ -680,6 +680,13 @@ interface JQueryCoordinates {
 }
 
 /**
+ * The interface used to specify the properties parameter in css()
+ */
+interface JQueryCssProperties {
+	[propertyName: string]: string | number | Function;
```

Comment:
What kind of `Function`? Could you express it like `(arg: ArgType, anotherArg: Arg2Type) => ReturnType`?